examples/simple_ipdb.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
```

Comment:
only one example is needed, right? two that do the same stuff is confusing, since not clear which one is recommended to use. Could you delete one that not recommended?